PolySTAR-Taproot-project/src/subsystems/turret/turret_test_auto_aim_command.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef TURRET_TEST_AUTO_AIM_COMMAND_HPP_
+#define TURRET_TEST_AUTO_AIM_COMMAND_HPP_
+
+#include "generic_auto_aim_command.hpp"
+
+namespace control
+{
+namespace turret
+{
+class TurretTestAutoAimCommand : public GenericAutoAimCommand
+{
+public:
+    /**
```

Comment:
Review comments to adapt it for the correct file. Should be turret and not chassis